.github/workflows/external_link_check.yml

```diff
@@ -34,9 +34,10 @@ jobs:
           restore-keys: cache-lychee- # restore latest cache (not a specific commit)
 
       - name: Check links
-        uses: lycheeverse/lychee-action@v2.0.2
+        uses: lycheeverse/lychee-action@v2
         with:
           args: --config './lychee.toml' --no-progress 'book/website/_build/html/**/*.html'
+          fails: false
```

Comment:
I'd have assumed we'd want it to fail on error, no?

https://github.com/lycheeverse/lychee-action?tab=readme-ov-file#passing-arguments